.github/workflows/pytest.yml

```diff
@@ -58,10 +58,17 @@ jobs:
     #       ${{ runner.os }}-
 
     # Install clang and it's python inteface via apt
-    - name: Install libclang-10
-      run: sudo apt-get update -y && sudo apt-get install -y libclang-10-dev
-    - name: Install python3-clang-10
-      run: sudo apt-get update -y && sudo apt-get install -y python3-clang-10
+    - name: Add llvm keys
+      run: |
+        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+        echo 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
+        echo 'deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
+    - name: Update cache
+      run: sudo apt-get update -y
+    - name: Install libclang-11
+      run: sudo apt-get install -y libclang-11-dev
+    - name: Install python3-clang-11
+      run: sudo apt-get install -y python3-clang-11
```

Comment:
If this ends up being the libclang used, I'd have only one step here (the purpose is installing `python3-clang-11`, the rest is ancillary/no value added), so only "Install python3-clang-11" instead of the three "Update cache", "Install libclang-11" and "Install python3-clang-11".

Reply:
Sure, done.